nmosconnection/api.py

```diff
@@ -56,64 +61,80 @@ def __init__(self, logger):
         self.schemaPath = SCHEMA_LOCAL
         self.useValidation = True  # Used for unit testing
 
-    def addSender(self, sender, uuid):
-        if uuid in self.senders:
+    def addSender(self, sender, senderId):
+        if senderId in self.senders:
             raise DuplicateRegistrationException(
-                "Sender already registered with uuid " + uuid
+                "Sender already registered with uuid " + senderId
             )
-        self.senders[uuid] = sender
-        self.activators[uuid] = Activator([sender])
-        return self.activators[uuid]
+        self.senders[senderId] = sender
+        self.activators[senderId] = Activator([sender])
+        return self.activators[senderId]
 
-    def addReceiver(self, receiver, uuid):
-        if uuid in self.receivers:
+    def addReceiver(self, receiver, receiverId):
+        if receiverId in self.receivers:
             raise DuplicateRegistrationException(
-                "Receiver already registered with uuid " + uuid
+                "Receiver already registered with uuid " + receiverId
             )
-        self.receivers[uuid] = receiver
+        self.receivers[receiverId] = receiver
         # Note the transport managers must be listed first so that they activate first
         # This ensures that SDP files are available via the API before any interactions with the driver
         if receiver.legs == 1:
-            self.activators[uuid] = Activator([
+            self.activators[receiverId] = Activator([
                 receiver.transportManagers[0],
                 receiver
             ])
         else:
-            self.activators[uuid] = Activator([
+            self.activators[receiverId] = Activator([
                 receiver.transportManagers[0],
                 receiver.transportManagers[1],
                 receiver
             ])
-        self.transportManagers[uuid] = receiver.transportManagers[0]
-        return self.activators[uuid]
+        self.transportManagers[receiverId] = receiver.transportManagers[0]
+        return self.activators[receiverId]
 
-    def getDevice(self, api_version, sr, device):
+    def getTransceiver(self, api_version, sr, srID):
```

Comment:
Any chance you could improve the 'sr' name here too? transceiverType? and transceiverID?